base_gps_pitch_tf: 0.0
base_gps_roll_tf: 0.0
base_gps_x_tf: 0.2
base_gps_y_tf: 0.0
base_gps_yaw_tf: -1.737464887445537
base_gps_z_tf: 0.25
